src/apps/cli/commands/add.py

```diff
@@ -448,4 +448,7 @@ def update_name(mainPackageDirName):
         defaultNix.write(template)
         print(f"Created {output}/default.nix")
 
-    print(f"Created {output}/dream-lock.json")
\ No newline at end of file
+    print(f"Created {output}/dream-lock.json")
+
+    if config['isRepo']:
+      sp.run(f"git add -N {output}".split())
```

Comment:
It would be better to pass a list when running commands to avoid quoting issues.

Reply:
Thanks. Will include this into my next PR